stac_pydantic/shared.py

```diff
@@ -41,6 +41,7 @@ class MimeTypes(str, Enum):
     html = "text/html"
     text = "text/plain"
     openapi = "application/vnd.oai.openapi+json;version=3.0"
+    schemajson = "application/schema+json"
```

Comment:
```suggestion
    jsonschema = "application/schema+json"
```
Might be a bit more intuitive?